pv_site_api/main.py

```diff
@@ -273,22 +273,44 @@ def post_pv_actual(
     session.commit()
 
 
-# Comment this out, until we have security on this
-# # put_site_info: client can update a site
-# @app.put("/sites/{site_uuid}")
-# def put_site_info(site_info: PVSiteMetadata):
-#     """
-#     ### This route allows a user to update site information for a single site.
-#
-#     """
-#
-#     if is_fake():
-#         print(f"Successfully updated {site_info.model_dump()} for site {site_info.client_site_name
-# }")
-#         print("Not doing anything with it (yet!)")
-#         return
-#
-#     raise Exception(NotImplemented)
+# put_site_info: client can update a site
+@app.put("/sites/{site_uuid}")
+def put_site_info(
+    site_uuid: str,
+    site_info: PVSiteEditMetadata,
+    session: Session = Depends(get_session),
+    auth: dict = Depends(auth),
```

Comment:
dict, should be `auth` so that in line 306 the it can passed to verify is the user has access to the site